woocommerce/wc-admin
# Migrate reports data store to wp data

Fixes #3377

Migrates the existing reports data store in wc-api to the data package using wp data.

### Detailed test instructions:

1. Go through all reports under Analytics.
1. Make sure the expected items and stats are received.
1. Try changing filters or pages to make sure items/stats are updated correctly.
1. Optionally, add an `wp_die()` to any of the report API endpoints.
1. Note that the error message appears as expected on the frontend.
